### PR TITLE
Add compat data for SpeechSynthesisUtterance

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/SpeechSynthesisUtterance

### CODE REVIEW

File: api/SpeechSynthesisUtterance.json [hunk 1]
```diff
@@ -0,0 +1,676 @@
+{
+  "api": {
+    "SpeechSynthesis": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SpeechSynthesisUtterance",
+        "support": {
+          "webview_android": {
+            "version_added": true
```
Comment (jpmedley) [inline api/SpeechSynthesisUtterance.json]: This is also `"33"`.

File: api/SpeechSynthesisUtterance.json [hunk 2]
```diff
@@ -0,0 +1,676 @@
...
+          },
+          "chrome": {
+            "version_added": "33"
+          },
+          "chrome_android": {
+            "version_added": true
```
Comment (jpmedley) [inline api/SpeechSynthesisUtterance.json]: 33

File: api/SpeechSynthesisUtterance.json [hunk 3]
```diff
@@ -0,0 +1,676 @@
...
+            "version_added": "49"
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": null
```
Comment (jpmedley) [inline api/SpeechSynthesisUtterance.json]: 21

File: api/SpeechSynthesisUtterance.json [hunk 4]
```diff
@@ -0,0 +1,676 @@
...
+          },
+          "opera_android": {
+            "version_added": false
```
Comment (jpmedley) [inline api/SpeechSynthesisUtterance.json]: 21

File: api/SpeechSynthesisUtterance.json [hunk 5]
```diff
@@ -0,0 +1,676 @@
...
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "charIndex": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SpeechSynthesisUtterance/lang",
+          "support": {
+            "webview_android": {
+              "version_added": true
```
Comment (jpmedley) [inline api/SpeechSynthesisUtterance.json]: 33

File: api/SpeechSynthesisUtterance.json [hunk 6]
```diff
@@ -0,0 +1,676 @@
...
+            },
+            "chrome": {
+              "version_added": "33"
+            },
+            "chrome_android": {
+              "version_added": true
```
Comment (jpmedley) [inline api/SpeechSynthesisUtterance.json]: 33

File: api/SpeechSynthesisUtterance.json [hunk 7]
```diff
@@ -0,0 +1,676 @@
...
+              "version_added": "49"
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SpeechSynthesisUtterance.json]: 21

File: api/SpeechSynthesisUtterance.json [hunk 8]
```diff
@@ -0,0 +1,676 @@
...
+            },
+            "opera_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/SpeechSynthesisUtterance.json]: 21

I'm being interrupted by airplane boarding. 33 for Chrome and 21 for Opera for all remaining members.

Comment (maboa) [reply api/SpeechSynthesisUtterance.json]: OK - I imagine this goes for this whole series of APIs - have a great flight!

**Pull request merged**